exist-core/src/main/java/org/exist/xquery/functions/fn/FunAnalyzeString.java

```diff
@@ -151,14 +151,10 @@ private void analyzeString(final MemTreeBuilder builder, final String input, Str
             }
         } catch (final net.sf.saxon.trans.XPathException e) {
             switch (e.getErrorCodeLocalPart()) {
-                case "FORX0001":
-                    throw new XPathException(this, ErrorCodes.FORX0001, e.getMessage());
-                case "FORX0002":
-                    throw new XPathException(this, ErrorCodes.FORX0002, e.getMessage());
-                case "FORX0003":
-                    throw new XPathException(this, ErrorCodes.FORX0003, e.getMessage());
-                default:
-                    throw new XPathException(this, e.getMessage());
+                case "FORX0001" -> throw new XPathException(this, ErrorCodes.FORX0001, e.getMessage());
+                case "FORX0002" -> throw new XPathException(this, ErrorCodes.FORX0002, e.getMessage());
+                case "FORX0003" -> throw new XPathException(this, ErrorCodes.FORX0003, e.getMessage());
+                default -> throw new XPathException(this, e.getMessage());
```

Comment:
Should this match the change in https://github.com/eXist-db/exist/pull/4886/commits/5f974298fa6f8eb83d654fe9c9f8f941200909b4#diff-b83837e14cf949b3eff92fc056203d1a62e5f5c444fb8891703e1b220c70e4e5R144 ? (I see `ErrorCodes.ERROR` is not present here.)

Reply:
Oh, good catch!